code/modules/spells/aoe_turf/conjure/forcewall.dm

```diff
@@ -51,3 +51,17 @@
 	new /obj/effect/forcefield/cult(get_turf(src))
 	qdel(src)
 	return
+
+/obj/effect/forcefield/invisible_wall
+	name = "invisible wall"
+	desc = "The nemesis of explorers since the dawn of times."
+
+/obj/effect/forcefield/invisible_wall/New()
+	..()
+
+	alpha = 0
```

Comment:
I believe invisibility is irrelevant to the mapper, so removing alpha and setting invisibility to max would do the job.